adapters/criteo/criteo.go

```diff
@@ -93,6 +109,21 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.R
 		}
 	}
 
+	if response.Ext != nil {
+		var responseExt CriteoExt
+		if err := json.Unmarshal(response.Ext, &responseExt); err == nil && responseExt.Igi != nil {
```

Comment:
The `responseExt.Igi != nil` check could evaluate to `true` even if `responseExt.Igi` is empty. In other words `responseExt.Igi` could be non-nil and still be empty. Should we modify this check to `len(responseExt.Igi) > 0` that covers both?

Reply:
Thanks, I added checks and tests on the edge cases.